website/docs/developer/development-guide/code/setup-dev-env.md

```diff
@@ -0,0 +1,61 @@
+---
+id: setup-dev-env
+title: Setting Up a Dev Environment
+description: Instructions on creating an environment for working with lambdas.
+---
+
+## Initial Setup
+1. git clone https://github.com/nasa/cumulus-orca
+1. Install [Python 3.8.X](https://www.python.org/downloads/) and restart if needed.
+   Make sure Python is added to your PATH.
+1. Install the AWS client. View the [official documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) if needed.
+    ```commandline
+    pip3 install awscli
+    ```
+
+## Per-Lambda Setup
+Lambdas should be built and tested within individual Virtual Environments.
+1. Navigate to the task's folder within 'culumus-orca\tasks\[task name]'.
+1. Create a new Virtual Environment and enter it.
```

Comment:
Possibly change from `Create a new Virtual Environment and enter it.` to `Create and activate a new Python virtual environment.`

Reply:
Updated.